### PR TITLE
Deleting a recording

### PR DESCRIPTION
**Pull request by Tanesan**
I created the ability to delete recorded videos and
Statefulwedget was used.
Then I got this error and it didn't work.
I tried to look up the error, but I can't figure it out, so please let me know.
Maybe, but I think I had to change some other part of the code since I used statefulwedget, but I don't know where to change it.

```
════════ Exception caught by rendering library ═════════════════════════════════════════════════════
'package:flutter/src/rendering/sliver_multi_box_adaptor.dart': Failed assertion: line 545 pos 12: 'child.hasSize': is not true.
The relevant error-causing widget was: 
  ListView file:///Users/keitotanemura/Desktop/job_interview_practice/lib/feature/recordings/presentation/pages/recordings_page.dart:26:31
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
Null check operator used on a null value
The relevant error-causing widget was: 
  ListView file:///Users/keitotanemura/Desktop/job_interview_practice/lib/feature/recordings/presentation/pages/recordings_page.dart:26:31
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
Null check operator used on a null value
The relevant error-causing widget was: 
  ListView file:///Users/keitotanemura/Desktop/job_interview_practice/lib/feature/recordings/presentation/pages/recordings_page.dart:26:31
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
Null check operator used on a null value
The relevant error-causing widget was: 
  ListView file:///Users/keitotanemura/Desktop/job_interview_practice/lib/feature/recordings/presentation/pages/recordings_page.dart:26:31
════════════════════════════════════════════════════════════════════════════════════════════════════

```

### CODE REVIEW

File: lib/feature/recordings/presentation/pages/recordings_page.dart [hunk 1]
```diff
@@ -24,13 +31,43 @@ class RecordingsPage extends StatelessWidget {
...
-                    title: Text(item.title),
-                    subtitle: Text("Answered on ${item.model.questions.length} questions"),
-                    trailing: Icon(Icons.play_circle_fill),
-                    onTap: () {
-                      Navigator.push(context, MaterialPageRoute(builder: (_) => RecodingDetailsPage(item)));
-                    },
+                      leading: Text(index.toString()),
+                      title: Text(item.title),
+                      subtitle: Text("Answered on ${item.model.questions.length} questions"),
+                      trailing: Scaffold(
```
Comment (elvi117) [inline lib/feature/recordings/presentation/pages/recordings_page.dart]: scaffold is not necessary here

File: lib/feature/recordings/presentation/pages/recordings_page.dart [hunk 2]
```diff
@@ -24,13 +31,43 @@ class RecordingsPage extends StatelessWidget {
...
+                        builder: (BuildContext context, AsyncSnapshot<Tag> snapshot) {
+                      if (mapController.ready &&
+                        snapshot.hasData &&
+                        snapshot.data.mobile.nid != 0) {
+                        tag = snapshot.data;
+                        widget.tagStream.sink.add(Tag());
+                  }
+                  }
+                  },
+                  ),
```
Comment (elvi117) [inline lib/feature/recordings/presentation/pages/recordings_page.dart]: whole this new code is totally wrong 😢 
example solution:
```dart
PopupMenuButton(
                            itemBuilder: (context) => [
                                  PopupMenuItem(
                                    value: 1,
                                    child: Text(
                                      'Play',
                                    ),
                                  ),
                                  PopupMenuItem(
                                    value: 2,
                                    child: Text('Delete'),
                                  ),
                                ],
                            onSelected: (index) {
                              switch (index) {
                                case 1:
                                  // Play action
                                  break;
                                case 2:
                                  // delete acition
                                  break;
                              }
                            },
                            child: Icon(
                              Icons.more_vert,
                              size: 28,
                              color: Colors.white,
                            ))```

**Review by elvi117 — CHANGES_REQUESTED** [review]

File: lib/feature/recordings/presentation/pages/recordings_page.dart (hunk 2 above)
Comment (Tanesan) [reply lib/feature/recordings/presentation/pages/recordings_page.dart]: I understood it.

**Comment by Tanesan** [comment]
I was able to fix it thanks to your advice. But now I don't know how to delete the list.

File: lib/feature/recordings/presentation/pages/recordings_page.dart (hunk 1 above)
Comment (Tanesan) [reply lib/feature/recordings/presentation/pages/recordings_page.dart]: I changed it !!

**Comment by Tanesan** [comment]
I don't know how to write the deletion the list in the Bloc anymore, do I create a new block class?

**Comment by Tanesan** [comment]
#35

**Pull request closed without merging**